### PR TITLE
Update Google cloud pro page #WD-1968

### PR DESCRIPTION
**Pull request by carkod**
## Done

Updated Google cloud pro pages.

## QA

- Compare https://ubuntu-com-12551.demos.haus/gcp/pro with [copy docs](https://docs.google.com/document/d/1zR1m6CH1POU5tnzNzgNYanG2_EGQtKVNR3_dPew1pf4/edit)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-1968?atlOrigin=eyJpIjoiNjI5ZjU5YzI3OGY2NDRiY2I0OTQzMmEwYmMwZGIwZGEiLCJwIjoiaiJ9

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

**Comment by tbille** [comment]
Added blocked label until this comment gets resolved: https://docs.google.com/document/d/1zR1m6CH1POU5tnzNzgNYanG2_EGQtKVNR3_dPew1pf4/edit#

File: templates/gcp/pro.html [hunk 1]
```diff
@@ -5,12 +5,17 @@
...
 {% block content %}
-<section class="p-strip--suru-topped u-no-padding--bottom">
+<section class="p-strip--suru-topped is-bordered">
   <div class="row u-equal-height">
     <div class="col-7 col-medium-4">
       <h1>Ubuntu Pro for Google Cloud</h1>
       <p class="p-heading--3">For production. For professionals.</p>
       <p>Ubuntu Pro for GCP is a premium image delivering the most comprehensive security and GCP compliance features. Ubuntu Pro is suitable for small and large-scale Linux enterprise operations offering pay-as-you-go billing on your existing GCP invoice.</p>
+      <h3 class="p-heading--4">Ubuntu Pro 22.04 LTS
+        <span class="p-status-label--rounded">Latest</span>
```
[the remark this answers is not in the document]
Comment (tbille) [reply templates/gcp/pro.html:15]: ok you can remove that Latest section.
The person who made the request came back to us, I am unblocking the PR.

**Comment by carkod** [comment]
> Added blocked label until this comment gets resolved: https://docs.google.com/document/d/1zR1m6CH1POU5tnzNzgNYanG2_EGQtKVNR3_dPew1pf4/edit#

Can you re-review when you can?

**Review by tbille — APPROVED** [review]
Looks good!

**Pull request merged**